package.json

```diff
@@ -32,8 +32,8 @@
     "postinstall": "test -n \"$SKIP_BUILD_INTERNAL_DEPENDENCIES\" || yarn build:internal-dependencies",
     "refresh-database": "yarn workspace @tupaia/database refresh-database --root ../../",
     "test-all": "node ./scripts/node/testAll --silent",
-    "update-codeship-env-vars": "./packages/devops/scripts/ci/updateCiEnvVars.sh",
```

Comment:
The `./packages/devops/scripts/ci/updateCiEnvVars.sh` script no longer exists, so I removed this command

Reply:
Thanks for tidying up!